tests/components/command_line/test_sensor.py

```diff
@@ -157,29 +147,25 @@ def test_update_with_missing_json_attrs(self):
         )
         self.sensor.update()
         assert "some_json_value" == self.sensor.device_state_attributes["key"]
-        assert (
-            "another_json_value" == self.sensor.device_state_attributes["another_key"]
-        )
-        assert "value_three" == self.sensor.device_state_attributes["key_three"]
+        assert ("another_json_value" ==
+                self.sensor.device_state_attributes["another_key"])
+        assert "value_three" == self.sensor.device_state_attributes[
+            "key_three"]
         assert not ("special_key" in self.sensor.device_state_attributes)
 
     def test_update_with_unnecessary_json_attrs(self):
         """Test attributes get extracted from a JSON result."""
         data = command_line.CommandSensorData(
             self.hass,
-            (
-                'echo { \\"key\\": \\"some_json_value\\", \\"another_key\\":\
-             \\"another_json_value\\", \\"key_three\\": \\"value_three\\" }'
-            ),
+            ('echo { \\"key\\": \\"some_json_value\\", \\"another_key\\":\
+             \\"another_json_value\\", \\"key_three\\": \\"value_three\\" }'),
             15,
         )
 
-        self.sensor = command_line.CommandSensor(
-            self.hass, data, "test", None, None, ["key", "another_key"]
-        )
+        self.sensor = command_line.CommandSensor(self.hass, data, "test", None,
+                                                 None, ["key", "another_key"])
         self.sensor.update()
         assert "some_json_value" == self.sensor.device_state_attributes["key"]
-        assert (
-            "another_json_value" == self.sensor.device_state_attributes["another_key"]
-        )
+        assert ("another_json_value" ==
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)